api/settings.go

```diff
@@ -352,7 +352,8 @@ const (
 	// Default: ""
 	IndentJSON = "INDENT_JSON"
 
-	// DevUseInsecureCookie wether to use a cookie with Secure set to false in development.
+	// DevUseInsecureCookie controls whether to use a cookie with Secure set to false in development.
```

Comment:
I think this makes sense to combine these two behaviors behind the one env var. I think we should probably rename it though. How about DEV_DISABLE_SSL ? 

You'll need to update this both in settings.go and in docs/CONFIGURE.md

Finally, I think that we should use this new environment variable in http/server.go to switch wether we are configuring TLS, instead of just checking to see if the TLS_{CERT,KEY} variables are present. We should throw an error then if DEV_DISABLE_SSL is not set but TLS_CERT is not present.

Reply:
Fixed